.coveragerc

```diff
@@ -1,11 +1,22 @@
 [run]
-branch = True
+parallel = true
+source =
+    pyramid_openapi3
+;branch = True
+
+[paths]
+source =
+    pyramid_openapi3
+    */site-packages/pyramid_openapi3
 
 [report]
-# Regexes for lines to exclude from consideration
-exclude_lines =
-    # Have to re-enable the standard pragma
-    pragma\: no cover
+show_missing = true
+precision = 2
 
-    # Don't complain if non-runnable code isn't run:
-    if __name__ == "__main__"\:
+# Regexes for lines to exclude from consideration
+;exclude_lines =
+;    # Have to re-enable the standard pragma
+;    pragma\: no cover
+;
+;    # Don't complain if non-runnable code isn't run:
+;    if __name__ == "__main__"\:
```

Comment:
These lines are needed because I don't want coverage to report `if __name__ == "__main__"` lines as not covered: writing tests for them is very cumbersome, and they are mostly one-liners.

Reply:
Done in cb5beaf